app/views/users/show.html.erb

```diff
@@ -1 +1,51 @@
-<h2>Show a particular user</h2>
\ No newline at end of file
+<div class="mt-3 d-flex align-items-center flex-column">
+  <div class="photo-name">
+    <div class="card mb-3 border-primary" style="width: 180px">
+      <img src="<%= @user.photo %>" alt="avatar" />
+    </div>
+    <div class="card-body">
+      <h2><%= @user.name %></h2>
+    </div>
+    <div class="d-flex justify-content-end" style="width: 350px">
+      <p class="card-text">
+        <small class="text-muted"
+          >Number of posts: <%= @user.posts_counter %></small
```

Comment:
- [ ] kindly change `posts_counter` to `postsCounter` as it is what you used in your schema